src/Components/NotificationDropdown.js

```diff
@@ -0,0 +1,108 @@
+import React, { Fragment, Component } from "react";
+import { withRouter } from 'react-router';
+import { MDBDropdown, MDBDropdownToggle, MDBDropdownMenu, MDBDropdownItem } from "mdbreact";
+import axios from "axios";
+import './styles/NotificationDropdown.css'
+
+function NotifIcon(props) {
+    const notiftype = props.value._notification_type;
+    const isread = props.value.is_read ? 'var(--medgray)' : 'var(--saturatedpurple)';
+    switch(notiftype) {
+        case 'EVENT_COMMUNICATION':
+            return <i class="fas fa-comment fa-2x" style={{color: isread}}></i>;
+        case 'EVENT_UPDATE':
+            return <i class="fas fa-calendar-day fa-2x" style={{color: isread}}></i>;   
+        case 'PEOPLE_MGMT':
+            return <i class="fas fa-users fa-2x" style={{color: isread}}></i>;
+        default:
+            return "";
+    }
+}
+
+class Notification extends Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            allRead: false,
+            notifications: []
+        };
+        this.isAllRead = this.isAllRead.bind(this);
+        this.isRead = this.isRead.bind(this);
+    }
+
+    componentDidMount() {
+        this.getNotifications();
+    }
+
+    isAllRead(){
+        let read = this.state.notifications.slice(0, 10).filter(e => e.is_read == false).length > 0 ? false : true;
```

Comment:
You don't need the ternary operator for it. Just add an exclamation mark and wrap the boolean expression in brackets. Would be cleaner